framework/doc/content/source/kernels/ADScaledCoupledTimeDerivative.md

```diff
@@ -0,0 +1,28 @@
+# ADScaledCoupledTimeDerivative
+
+## Description
+
+The `ADScaledCoupledTimeDerivative` kernel is an extension of the
+[`ADCoupledTimeDerivative`](/ADCoupledTimeDerivative.md) kernel that has a scaling factor.
+This factor can be:
+
+- an ADReal-valued material property using the [!param](/Kernels/ADScaledCoupledTimeDerivative/mat_prop) parameter
+
+The Jacobian contribution is computed using forward mode automatic
+differentiation.
```

Comment:
```suggestion
This factor can be an ADReal-valued material property, set using the [!param](/Kernels/ADScaledCoupledTimeDerivative/mat_prop)
parameter. The Jacobian contribution is computed using forward mode automatic differentiation.
```

Let's condense this since the list is not really a list.